specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -12183,22 +12230,6 @@
           "readOnly": true,
           "description": "Describes whether the operation ran out of time before it completed all its intended actions"
         },
-        "rebootStatus": {
-          "type": "string",
-          "readOnly": true,
-          "description": "The reboot status of the machine after the patch operation. It will be in \"NotNeeded\" status if reboot is not needed after the patch operation. \"Required\" will be the status once the patch is applied and machine is required to reboot. \"Started\" will be the reboot status when the machine has started to reboot. \"Failed\" will be the status if the machine is failed to reboot. \"Completed\" will be the status once the machine is rebooted successfully",
-          "enum": [
-            "NotNeeded",
-            "Required",
-            "Started",
-            "Failed",
-            "Completed"
-          ],
-          "x-ms-enum": {
-            "name": "RebootStatus",
-            "modelAsString": true
-          }
-        },
```

Comment:
Why has this been removed from swagger?

Reply:
It's no longer in the API's response model: https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=%2Fsrc%2FCRP%2FDev%2FComputeResourceProvider%2FApiEntityModel%2FVMGuestPatch%2FVMGuestPatchApiEntityModel.cs&version=GBmaster&line=122&lineEnd=122&lineStartColumn=18&lineEndColumn=54&lineStyle=plain&_a=contents